sources/AutoFilerTab.cpp

```diff
@@ -226,12 +235,14 @@ AutoFilerTab::MessageReceived(BMessage* msg)
 				break;
 
 			BStringItem* item = (BStringItem*)fFolderList->ItemAt(selection);
-			fFilePanel->SetPanelDirectory(item->Text());
+			fEditPanel->SetPanelDirectory(item->Text());
 
 			BMessage panelMsg(MSG_FOLDER_CHOSEN);
 			panelMsg.AddInt32("index", selection);
-			fFilePanel->SetMessage(&panelMsg);
-			fFilePanel->Show();
+			fEditPanel->SetMessage(&panelMsg);
+			fEditPanel->Window()->SetTitle("AutoFiler: Edit folder");
+			fEditPanel->SetButtonLabel(B_DEFAULT_BUTTON, "Choose");
```

Comment:
I had all of them in the constructor at first, but it caused one panel's buttons to interfere with the other's and vice versa, probably due to bugs in BFilePanel. Now that we are going to remove the Edit button from the AutoFiler tab, the problem will no longer exist. I will move them back to the constructor when removing the Edit button.